src/unofficial.md

```diff
@@ -19,6 +19,7 @@ Application domains:
 * [Command Line Applications in Rust](https://rust-cli.github.io/book/index.html)
 * [Async programming in Rust with async-std](https://book.async.rs/introduction.html)
 * [The Embedded Rust Book](https://rust-embedded.github.io/book/)
+* [The Embedded Rust Discovery Book](https://docs.rust-embedded.org/discovery/) - introduction to embedded systems using Rust
```

Comment:
I'm taking care to use the literal title of the book (see the README), which is admittedly not great in this case:
```suggestion
* [Discovery](https://docs.rust-embedded.org/discovery/) - introduction to embedded systems using Rust
```